content_development/MAIN.md

```diff
@@ -68,9 +68,9 @@ Some of common advantages for developers include:
 
 As such, the main advantages for researchers (users) include lower costs, increased transparency, increased security and stability, no vendor 'lock in' with increased user control, and overall higher quality. Furthermore, sharing OSS allows researchers to receive credit for their efforts, for example through direct software citation [(Smith et al., 2016)](https://github.com/OpenScienceMOOC/Module-5-Open-Research-Software-and-Open-Source/blob/master/Reading%20Material_Open%20Source%20and%20Open%20Research%20Software/Smith%20et%20al.%2C%202016.pdf).
 
-Commonly used OSS services include the [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/) internet browser and the [LibreOffice](https://www.libreoffice.org/) full office suite. LibreOffice is similar to the popular Microsoft Office, including a word processor, spreadsheet manager, and slide presentation software, but is completely free and Open Source.
+Commonly used OSS include the [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/) internet browser and the [LibreOffice](https://www.libreoffice.org/) full office suite. LibreOffice is similar to the popular Microsoft Office, including a word processor, spreadsheet manager, and slide presentation software, but is completely free and Open Source.
 
-Some regard the OSS movement to represent a counter-movement to neoliberalism and privatisation, through defiance of regulations and norms in the construction and re-use of information, and a potential transformation of modern-day capitalism through making software abundantly available with minimal effort.
+Some regard the OSS movement to represent a counter-movement to neoliberalism and privatisation, through defiance of regulations and norms in the construction and re-use of information, and a potential transformation of modern-day capitalism through making software abundantly available with minimal effort. <!-- reference? -->
```

Comment:
TODO

Reply:
Maybe here http://www.redalyc.org/html/742/74212712006/ ?

Reply:
(BTW the sentence is quite hard to understand for non-native English speakers like me...)

Reply:
Danke! Hm, so that is a good point, and something I'm fully aware of. The intention ultimately is to have these modules translated into as many languages as possible for this reason. I don't want to discriminate at all. However, mein Deutsch ist nicht sehr gut, and so I have to start in English for this reason! I'll try and simplify though.

Reply:
I wouldn't go into translation of everything, when it is not clear that there enough people for continuous maintenance. It is much easier to maintain one language version and English is usually fine for that.

> I'll try and simplify though.

Thank you, that is appreciated. I haven't read that paper, otherwise I could try to simplify it myself.

Reply:
I see what you mean. But I want this to be as open as possible to people with different language preferences and abilities. The power of doing these things in releases is that changes for translations become relatively simple. In theory.. That being said, the whole thing still needs a copy edit to make sure it reads nicely.